### PR TITLE
website: ButtonGroup

### PR DESCRIPTION
**Pull request by gretanausedaite**
## Changes

Updated ButtonGroup documentation page
Solves ButtonGroup part of #961

## Testing

N/A

## Docs

N/A

### CODE REVIEW

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 1]
```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
...
+
+Use <code>orientation</code> prop to switch button group to vertical.
+
+<LiveExample src='ButtonGroup.vertical.tsx'>
+  <AllExamples.ButtonGroupVerticalExample client:load />
+</LiveExample>
+
+### With input
+
+You can combine inputs with buttons in a button group.
```
Comment (FlyersPh9) [inline apps/website/src/pages/docs/buttongroup.mdx:45]:
```suggestion
You can combine [inputs](input) with buttons in a button group.
```

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: Button group
-description: Buttons allow users to take actions and make choices with a single tap or click.
+description: Buttons group is a set of buttons with similar functionality.
```
Comment (adamhock) [inline apps/website/src/pages/docs/buttongroup.mdx:3]: nit

```suggestion
description: A button group is a set of buttons with similar functionality.
```

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 3]
```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
...
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='buttons-buttongroup' />
-
 <LiveExample src='ButtonGroup.main.tsx'>
   <AllExamples.ButtonGroupMainExample client:load />
 </LiveExample>
 
-A button group consists of similar tools. The button group can only consist of default buttons or borderless buttons, a button group can not be a mix of the two. If there is not enough space to display all of the buttons, the buttons can truncate into an overflow menu.
+Use a button group if you want to but similar functionality in one place. It helps users differentiate functional areas of the UI.
```
Comment (adamhock) [inline apps/website/src/pages/docs/buttongroup.mdx:21]:
```suggestion
Use a button group if you want to put similar functionality in one place. It helps users differentiate functional areas of the UI.
```

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 4]
```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
...
+
+Buttons in button group can be active, disabled and both, active and disabled.
```
Comment (adamhock) [inline apps/website/src/pages/docs/buttongroup.mdx:23]:
```suggestion
Buttons in a button group can be active, disabled or both active and disabled.
```

or 

`Buttons in a button group can be active, disabled and both.`

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 5]
```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
...
+
+If there is not enough space to display all of the buttons, the buttons can truncate into an overflow menu.
+
+<LiveExample src='ButtonGroup.overflow.tsx'>
+  <AllExamples.ButtonGroupOverflowExample client:load />
+</LiveExample>
+
+### Vertical
+
+Use <code>orientation</code> prop to switch button group to vertical.
```
Comment (adamhock) [inline apps/website/src/pages/docs/buttongroup.mdx:37]:
```suggestion
Use the <code>orientation</code> prop to switch the button group to vertical.
```

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 6]
```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
...
+You can combine [inputs](input) with buttons in a button group.
+
+<LiveExample src='ButtonGroup.input.tsx'>
+  <AllExamples.ButtonGroupInputExample client:load />
+</LiveExample>
+
+## Usage
+
+- The button group should only consist of default buttons or borderless buttons
+- You should not mix different styles buttons in the same group.
```
Comment (adamhock) [inline apps/website/src/pages/docs/buttongroup.mdx:54]:
```suggestion
- You should not mix buttons with different styles in the same group.
```

OR

`You should not mix different styled buttons in the same group.`

**Review by adamhock — COMMENTED** [review]
The examples and formatting of information all look good. I've added a few proofreading comments, but other than that it looks good to me!

File: apps/website/src/pages/docs/buttongroup.mdx (hunk 6 above)
Comment (gretanausedaite) [reply apps/website/src/pages/docs/buttongroup.mdx]: Updated to OR version

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 7]
```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
...
+
+You can combine [inputs](input) with buttons in a button group.
+
+<LiveExample src='ButtonGroup.input.tsx'>
+  <AllExamples.ButtonGroupInputExample client:load />
+</LiveExample>
+
+## Usage
+
+- The button group should only consist of default buttons or borderless buttons
```
Comment (FlyersPh9) [inline apps/website/src/pages/docs/buttongroup.mdx:53]:
```suggestion
- The button group should only consist of [default buttons](button#default) or [borderless buttons](button#borderless)
```
I think these links are correct based on #1158 but I did not confirm.

File: apps/website/src/pages/docs/buttongroup.mdx [hunk 8]
```diff
@@ -14,14 +14,50 @@ import * as AllExamples from '~/examples';
...
+
+If there is not enough space to display all of the buttons, the buttons can truncate into an overflow menu.
+
+<LiveExample src='ButtonGroup.overflow.tsx'>
+  <AllExamples.ButtonGroupOverflowExample client:load />
+</LiveExample>
+
+### Vertical
+
+Use the <code>orientation</code> prop to switch the button group to vertical.
```
Comment (FlyersPh9) [inline apps/website/src/pages/docs/buttongroup.mdx:37]: Does this not work?
```suggestion
Use the `orientation` prop to switch the button group to vertical.
```

**Review by FlyersPh9 — APPROVED** [review]

**Pull request merged**